src/commands/createNewProject/createNewProject.ts

```diff
@@ -106,3 +110,25 @@ export function getProjectCreator(language: string, functionAppPath: string, act
             return new ScriptProjectCreatorBase(functionAppPath, actionContext.properties);
     }
 }
+
+async function addToFilesExclude(filesToExclude: string | string[], fsPath: string): Promise<void> {
+    const projectConfiguration: WorkspaceConfiguration = workspace.getConfiguration('files', Uri.file(fsPath));
+    // tslint:disable:no-any no-unsafe-any
+    let excludedFiles: any | undefined = projectConfiguration.get('exclude');
+    if (excludedFiles) {
+        for (const key of Object.keys(DefaultFilesExcluded)) {
```

Comment:
I don't understand why you're doing this.

Reply:
I wanted to make the function a bit more versatile than just adding `obj` and `bin` into the files.exclude property, but wanted to allow users the option to either input just one directory as a string or multiple directories as a string[].

Reply:
No, I don't understand the code.